daprdocs/content/en/reference/components-reference/supported-pubsub/setup-confluent-kafka.md

```diff
@@ -0,0 +1,112 @@
+---
+type: docs
+title: "Apache Kafka (confluent kafka)"
+linkTitle: "Apache Kafka (confluent kafka)"
```

Comment:
```suggestion
linkTitle: "Apache Kafka (Confluent)"
```
In the linkTitle it's best to be as concise as possible. Would shortening to this make sense?